class_ids: ["pallet", "forklift", "worker", "door", "cart"]
